flux_local/manifest.py

```diff
@@ -141,9 +141,8 @@ def parse_doc(cls, doc: dict[str, Any], default_namespace: str) -> "HelmChart":
             if not (name := chart_ref.get("name")):
                 raise InputException(f"Invalid {cls} missing spec.chartRef.name: {doc}")
             if not (namespace := chart_ref.get("namespace")):
```

Comment:
How about modifying line 150 to be `repo_namespace=chart_ref.get("namespace", default_namespace),` to make it exactly the same as the same as line 168.

Reply:
Yep that's better.